api/.htaccess

```diff
@@ -0,0 +1,7 @@
+RewriteEngine on
+RewriteCond %{REQUEST_FILENAME} !-d
+RewriteCond %{REQUEST_FILENAME} !-f
+RewriteRule . index.php [L]
+<ifmodule mod_deflate.c>
+AddOutputFilterByType DEFLATE application/json
+</ifmodule>
```

Comment:
This .htaccess was used during development, since the documented ([development setup](https://github.com/glidernet/ogn-ddb/blob/master/INSTALL.md#development-setup)) contained an apache (just for dev). If I finish this PR, I will replace this .htaccess file with a note in the README.